.gitignore

```diff
@@ -114,6 +114,9 @@ package-lock.json
 /packages/react-native/template/ios/Podfile.lock
 /packages/rn-tester/Gemfile.lock
 
+# RNTester sourcemaps [macOS]
+/packages/rn-tester/sourcemap.*.map
+
```

Comment:
These have started showing up? :o

Reply:
Yep, because we specify `SOURCEMAP_FILE` in our "Build JS bundle" build step.

Reply:
Hmm, iOS has done this for a while. Why didn't upstream need this ignore entry?

Reply:
The sourcemap gets generated for release builds only unless we set `FORCE_BUNDLING=true` in the Build JS Bundle script. We have separate schemes for building release builds of RNTester, but upstream doesn't.

Reply:
Hmm, ok. Upstream seems still able to test debug and release, so I wonder if we can remove our extra schemes and get rid of a diff. Not blocking though.

Reply:
https://github.com/microsoft/react-native-macos/pull/1915